src/utils/roofline_calc.py

```diff
@@ -306,8 +304,8 @@ def calc_ai(sort_type, ret_df):
             lds_data += (
                 (df["SQ_LDS_IDX_ACTIVE"][idx] - df["SQ_LDS_BANK_CONFLICT"][idx])
                 * 4
-                * L2_BANKS
-            )  # L2_BANKS = 32 (since assuming mi200 or mi300)
+                * (mspec.lds_banks_per_cu)
```

Comment:
Same number (32) in the end but should be lds banks- changing to using the per-arch defined lds_banks_per_cu from soc file instead of hardcoded value.